src/routes/wohnen/containers/WohnenExtraPage.js

```diff
@@ -62,15 +63,14 @@ export class WohnenExtraPage extends React.Component<PropsType> {
 
       return (
         <>
-          <Helmet title={`${offer.formData.accommodation.title} - ${extra.title} - ${cityName}`} />
+          <Helmet getPageTitle={this.getPageTitle(offer)} />
```

Comment:
is this needed?

Reply:
It isn't, I think its easier to handle this here though. For the sake of consistency we could also move it to the routeConfig, what do you prefer?